clients/flash/air-client/src/org/bigbluebutton/air/main/views/MainView.as

```diff
@@ -80,7 +85,11 @@ package org.bigbluebutton.air.main.views {
 			
 			_webcamDock.bottom = _menuButtons.height;
 			
-			_pollButton.bottom = getStyle("pollPadding");;
+			_pollButton.bottom = _menuButtons.height - _pollButton.height - 8;
+		}
+		
+		public function showPollingButton(visible:Boolean):void {
+			_pollButton.visible = _pollButton.includeInLayout = visible;

```

Comment:
MainView uses BasicLayout and I don't think setting includeInLayout applies with that layout.

Reply:
Same as previous remark.